src/Symfony/Component/Profiler/ProfileData/TimeData.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Profiler\ProfileData;
+use Symfony\Component\Stopwatch\Stopwatch;
```

Comment:
is Stopwatch component an hard dependency ? In my opinion it shouldn't. In all cases, this need to be explicit in the composer.json file